values.yaml

```diff
@@ -201,6 +201,8 @@ airflow:
     {{- $nodeSelector := or .Values.nodeSelector .Values.workers.nodeSelector }}
     {{- $affinity := or .Values.affinity .Values.workers.affinity }}
     {{- $tolerations := or .Values.tolerations .Values.workers.tolerations }}
+    {{- $securityContext := include "airflowPodSecurityContext" (list . .Values.workers) }}
```

Comment:
Shouldn't this match the style of the line below and be:
```suggestion
    {{- $airflowPodSecurityContext := include "airflowPodSecurityContext" (list . .Values.workers) }}
```

Reply:
Where is this `airflowPodSecurityContext` template?

Reply:
we are using airflow oss chart mappings here

Reply:
yes if this naming suits we can rename it as long as we all know why is this being used here